CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 All notable changes to this project will be documented in this file.
 
+## [0.8.0] - 2020-04-XX
+
+- Added multiple env configuration (Debug, QA, Stage and Production) for Android and iOS projects
+- Removed unused targets from iOS project and Podfile (ruby script)
+- Removed temporary fix for the `update_bundle_identifier` Fastlane lane
```

Comment:
this is solved :D

Reply:
I'm going to fix this in the next PR (google-services script)